### DISCUSSION TITLE
Welcome to Hackintosh Discussions!

### DISCUSSION THREAD

**Discussion by daliansky** [daliansky/Hackintosh#254] [General]
## Hackintosh 机型讨论
> 请不要发布类似：求`xxx`机型之类的话题

**Comment by opaulU** [comment]
Hi everyone, is there any EFI file that can work on my Samsung Ativ Book 9 PC (250 SATA SSD, 2.2ghz, i5-5200u, and 5500 intel graphic)?
I want to dualboot win10 and bigsur.
